### ISSUE TITLE
Support newer compressions like xz or zstd

### ISSUE THREAD

**Issue by debuglevel** [phpmyadmin/phpmyadmin#16483]
### Is your feature request related to a problem? Please describe.
SQL dumps are often very big and uploading them takes a while. Compressions helps, but phpMyAdmin only support zip, gzip and bzip2.

### Describe the solution you'd like
phpMyAdmin should support some newer compression methods like `xz` or `zstd`.

### Describe alternatives you've considered
bzip2 already compresses my dumps to about 25%, but xz goes down to about 16%

### Additional context
- phpMyAdmin version: 5.0.4

**Comment by ibennetch** [comment]
Good idea. In my personal projects, I have long since replaced both gzip
and bzip2 with xz, so I definitely see the benefits of this.

**Comment by ibennetch** [comment]
Currently, there seems to be:
* [php-ext-xz](https://github.com/codemasher/php-ext-xz) which seems to require a somewhat complicated build process (but was forked from a repository of our own @dan11 !) and seems to be pretty active.
* [php-xz](https://github.com/payden/php-xz) which seems slightly easier to build, but doesn't look like it's been updated in years.
* [zstd](https://pecl.php.net/package/zstd) is part of PECL, which makes it the most promising at the moment, but I don't personally use a lot of zstd in my toolchain yet.

What we do for other export types is to test whether the functions exist and display the appropriate options in the dropdown if the compression format is present, so it should be pretty trivial for us to start supporting these within phpMyAdmin.

**Comment by williamdes** [comment]
https://github.com/kjdev/php-ext-zstd seems like a better supported option, also in terms of it available on package systems and on pecl

**Comment by thezoggy** [comment]
what about rar?

**Comment by williamdes** [comment]
> what about rar?

https://pecl.php.net/package/rar
https://stackoverflow.com/a/36914615/5155484

Seems like it should be possible too

**Comment by ibennetch** [comment]
> what about rar?

It seems to me that it isn't worth the effort to package a .rar. That
format isn't requested often and usually one of the other formats we offer
are just as suitable. Do you feel rar would give an advantage over our
other formats?

**Comment by thezoggy** [comment]
in my world rar/tar/gzip/zip are kinda the go to, i never see lzma/xz/zst. bzip2 would be great to add but depending on the platform may not work out of the box once the end user gets the file. 

anyways just seems like if your going to offer up tar/zip you might as well toss rar in that mix since imho its been around forever and trivial to deal with on most platforms.

but guess thats one way to look at this, what archive formats are the best (compression wise / most accessible / more security features or can handle better unicode or have built in checksum+recovery) where the focus may be to output a file that is reliable besides convenient
